src/Compatibility/Core/src/AppHostBuilderExtensions.cs

```diff
@@ -6,6 +6,15 @@ namespace Microsoft.Maui.Controls.Compatibility
 {
 	public static class AppHostBuilderExtensions
 	{
+		public static IAppHostBuilder RegisterCompatibilityForms(this IAppHostBuilder builder)
```

Comment:
This is the special place for now since we used to init things like `Device.Info` and `PlatformServices` in the `Forms.Init` but that is currently running inside the `Activity`. However, we are now creating apps form the `Application` area. 

Once we split all this stuff out of `Forms` and into services then this can be removed.